frontend/language/src/nb.json

```diff
@@ -181,6 +180,10 @@
   "code_list_editor.text_resource.label.select": "Finn ledetekst for verdi nummer {{number}}",
   "code_list_editor.text_resource.label.value": "Oppgi ledetekst for verdi nummer {{number}}",
   "code_list_editor.text_resource.no_text_resource_option_label": "Ikke oppgitt",
+  "code_list_editor.type_selector_label": "Type verdier",
```

Comment:
Ref. my previous comment, maybe we can change this to "Kodelisten er tom. Velg type verdien til kodelisten. Dette kan ikke endres etter at du har lagg til alternativer" or something? 😄 
Then we also get to inform the user that the select is removed once the user clicks the add button 😄

Reply:
I like it 😄